text/maps/SSAnne2F.asm

```diff
@@ -1,63 +1,64 @@
 _SSAnne2Text1::
-	text "This ship, she is"
-	line "a luxury liner"
-	cont "for trainers!"
+	text "הספינה הזאת היא"
+	line "ספינת יוקרה"
+	cont "למאמנים!"
 
-	para "At every port, we"
-	line "hold parties with"
-	cont "invited trainers!"
+	para "בכל נמל, אנחנו"
+	line "עורכים מסיבות"
+	cont "למאמנים!"
 	done
 
 _SSAnneRivalBeforeBattleText::
-	text "<RIVAL>: Bonjour!"
+	text "בונז'ור:<RIVAL>!"
 	line "<PLAYER>!"
 
-	para "Imagine seeing"
-	line "you here!"
+	para "לא ציפיתי לראות"
+	line "אותך כאן!"
 
-	para "<PLAYER>, were you"
-	line "really invited?"
+	para "באמת ,<PLAYER>"
+	line "הוזמנת לכאן?"
 
-	para "So how's your"
-	line "#DEX coming?"
+	para "אז איך הולך"
+	line "עם ה#ידע?"
 
-	para "I already caught"
-	line "40 kinds, pal!"
+	para "כבר תפסתי 40"
+	line "סוגים, חבר!"
 
-	para "Different kinds"
-	line "are everywhere!"
+	para "יש סוגים שונים"
+	line "בכל מקום!"
 
-	para "Crawl around in"
-	line "grassy areas!"
+	para "מסתובבים באיזורים"
+	line "עשביים!"
```

Comment:
```suggestion
	para "מזדחלים סביב"
	line "באיזורים עשביים!"
```

Reply:
השאלה היא אם מדובר רק בפוקימונים שזוחלים? חשבתי שזה יותר כללי... 
מה אומרת?

Reply:
אני מסכימה, נשמע לי יותר כללי. אולי פועל בסגנון "שורצים" או משהו אחר to that effect

Reply:
מסכימה עם נגה, אולי שורצים אולי מתחבאים. לשיקולך